src/logger.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include <filesystem>
+#include <fstream>
+#include <iostream>
+#include <locale>
+#include <sstream>
+
+namespace ohl::logger {
+
+
+/**
+ * @brief Logs a logMessage to the log file ohl::logger::logPath. \n will be
+ * added at the end
+ *
+ * @param logMessage the message to be logged.
+ */
+void log( const char * logMessage );
+void log( const wchar_t * logMessage );
+void log( const std::ostream& logMessage );
+void log( const std::wostream& logMessage );
+/**
+ * @brief Logs AND prints a logMessage to the log file ohl::logger::logPath. \n will be
+ * added at the end
```

Comment:
This should be the default, there's no use for a (public) function which doesn't send to all log consumers.

In a more complex example, the consumers might only output messages at certain log levels, but they should still be sent every one so that they can each make that call.